### PR TITLE
Refactor table initialization in testFromUtcTimestamp

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

**Review by hashhar — APPROVED** [review]

File: testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViews.java [hunk 1]
```diff
@@ -177,17 +177,30 @@ public void testLateralViewJsonTupleAs()
...
+                "  source_integer int, " +
+                "  source_bigint bigint, " +
+                "  source_float float, " +
+                "  source_double double, " +
+                "  source_decimal_three decimal(10,3), " +
+                "  source_decimal_zero decimal(10,0), " +
+                "  source_timestamp timestamp, " +
+                "  source_date date" +
+                ")");
+        onTrino().executeQuery("INSERT INTO test_from_utc_timestamp_source VALUES ( " +
```
Comment (hashhar) [inline testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViews.java:192]: nit: Maybe add a comment why to save someone from "improving" the code in the future?

File: testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViews.java [hunk 2]
```diff
@@ -211,7 +224,7 @@ public void testFromUtcTimestamp()
                 .containsOnly(row(
                         "1969-12-31 16:00:00.123",
                         "1969-12-31 16:00:10.123",
-                        "1969-12-11 22:57:12.827",
+                        "1970-01-03 16:00:00.123",
```
Comment (hashhar) [inline testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViews.java:227]: Would be easier to understand the motivation for this change if it was a preparatory commit.

**Review by hashhar — APPROVED** [review]
LGTM % comments if it works.

**Comment by losipiuk** [comment]
AC

**Review by hashhar — APPROVED** [review]

**Pull request merged**